Week5/MyFavoriteMovies/app/src/main/java/com/raywenderlich/myfavoritemovies/model/MovieDao.kt

```diff
@@ -0,0 +1,19 @@
+package com.raywenderlich.myfavoritemovies.model
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.Query
+
+@Dao
+interface MovieDao {
```

Comment:
One thing I don't understand is that you've created all these functions for data handling, but you haven't implemented a single thing in the UI, to support them.

You still just display the dummy data, and you don't even bother creating the Repository, let alone using it to fetch or store data.